test/0530-bz1667801-fix-kibana-replica-shards

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# test init script tests cleanup of old SG indices
+# https://bugzilla.redhat.com/show_bug.cgi?id=1658632
+LOGGING_NS=${LOGGING_NS:-openshift-logging}
+
+if [ -n "${DEBUG:-}" ] ; then
+    set -x
+fi
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+source "${OS_O_A_L_DIR}/hack/testing/util.sh"
+source "${OS_O_A_L_DIR}/elasticsearch/init/common"
+
+os::util::environment::use_sudo
+
+test_name=$(basename $0)
+init_script="0530-bz1667801-fix-kibana-replica-shards"
+os::test::junit::declare_suite_start ${test_name}
+
+cleanup() {
+    local return_code="$?"
+    set +e
+    if [ $return_code = 0 ] ; then
+        mycmd=os::log::info
+    else
+        mycmd=os::log::error
+    fi
+    $mycmd ${test_name} test finished at $( date )
+    # this will call declare_test_end, suite_end, etc.
+    os::test::junit::reconcile_output
+    exit $return_code
+}
+trap "cleanup" EXIT
+
+seed_data(){
+  os::log::info Seeding data for the test
+  local pod=$1
+  names=(abc123 xyz123 zzz123)
+  for n in "${names[@]}"
+  do
+    index=".kibana.$n"
+    os::log::info Creating index $index
+    result=$(oc -n ${LOGGING_NS} exec -c elasticsearch $pod -- es_util --query=$index -XPUT -d '{"settings": {"index":{"number_of_replicas":"2"}}}')
```

Comment:
you can use `--query=${index}?pretty` here to get the result in formatted JSON

Reply:
I don't think we need formatted output here as we strickly are looking to see if there was an error